### PR TITLE
RHELAI-838 Enforce insights registration in ilab wrapper

### PR DESCRIPTION
**Pull request by javipolo**
I'd also appreciate if a native english speaker could review the error message displayed

/cc @fabiendupont

### CODE REVIEW

**Comment by fabiendupont** [comment]
Looks good to me, but I'm not a native english speaker :)

File: training/ilab-wrapper/ilab [hunk 1]
```diff
@@ -18,6 +18,32 @@ function verify_range() {
     fi
 }
 
+check_insights() {
+    if [[ -f /etc/insights-client/machine-id ]]; then
+        return
+    fi
+    if [[ -f /etc/ilab/skip-insights-validation ]]; then
```
Comment (jeremyeder) [inline training/ilab-wrapper/ilab]: make this file name "insights-opt-out" so its more explicit what the user is doing.

File: training/ilab-wrapper/ilab [hunk 2]
```diff
@@ -18,6 +18,32 @@ function verify_range() {
...
+        return
+    fi
+    cat << EOF
+This host is not connected to Red Hat Insights.
+
+To connect this host to Red Hat Insights run the following command:
+sudo rhc connect --organization <org_id> --activation-key <your_activation_key>
+
+For more information managing your activation keys, please visit:
+https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_activation_keys_on_the_hybrid_cloud_console/assembly-creating-managing-activation-keys
```
Comment (jeremyeder) [inline training/ilab-wrapper/ilab:43]: That doc is very thorough.  Possibly too much for a new user.  Proposing this change:

```This host is not connected to Red Hat Insights.
To connect this host to Red Hat Insights run the following command:
sudo rhc connect --organization <org_id> --activation-key <your_activation_key>

To generate an Activation Key: 
https://console.redhat.com/insights/connector/activation-keys (this page will also display your Organization ID).

For more information on Red Hat Insights, please visit:
https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_activation_keys_on_the_hybrid_cloud_console/assembly-creating-managing-activation-keys```

**Review by jeremyeder — CHANGES_REQUESTED** [review]

File: training/ilab-wrapper/ilab (hunk 1 above)
Comment (javipolo) [reply training/ilab-wrapper/ilab]: Sounds good to me

File: training/ilab-wrapper/ilab (hunk 2 above)
Comment (javipolo) [reply training/ilab-wrapper/ilab:43]: I like your change

However, in the future we will provide a link to the official RHELAI documentation that is more concise, but for now, let's keep it as you suggest

**Comment by jeremyeder** [comment]
LGTM

File: training/ilab-wrapper/ilab (hunk 2 above)
Comment (jeremyeder) [reply training/ilab-wrapper/ilab:43]: awesome thanks

**Comment by rhatdan** [comment]
LGTM

**Review by rhatdan — APPROVED** [review]
LGTM

**Pull request merged**